Cargo.lock

```diff
@@ -3639,7 +3637,7 @@ dependencies = [
  "paperclip-actix",
  "paperclip-core",
  "paperclip-macros",
- "parking_lot 0.11.2",
+ "parking_lot 0.10.2",
```

Comment:
Really surprising to see the downgrade here.

Reply:
reverted, nice catch.